<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>941</width>
     <height>751</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background: qlineargradient(
        x1: 0, y1: 0, x2: 1, y2: 0,
        stop: 0 #2a1f4d,
        stop: 0.4 #3d2b6b,
        stop: 1 #1e1b2e
    );
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>390</y>
     <width>224</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 22px;
    font-weight: normal;
    color: #e0d4ff;
}</string>
   </property>
   <property name="text">
    <string>Confirm New Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_Button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>490</y>
     <width>450</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background: qlineargradient(
        x1: 0, y1: 0, x2: 1, y2: 0,
        stop: 0 #6a5acd, 
        stop: 1 #9d81ff
    );
    color: white;
    border: none;
    border-radius: 22px;
    padding: 8px;
    font-size: 29px;
    font-weight: bold;
}

QPushButton:hover {
    background: qlineargradient(
        x1: 0, y1: 0, x2: 1, y2: 0,
        stop: 0 #9d81ff, 
        stop: 1 #b19bff
    );
    box-shadow: 0 0 20px rgba(157, 129, 255, 0.6);
}

QPushButton:pressed {
    background: qlineargradient(
        x1: 0, y1: 0, x2: 1, y2: 0,
        stop: 0 #5a4acd, 
        stop: 1 #8d71ef
    );
}</string>
   </property>
   <property name="text">
    <string>Reset Password</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirm_new_password_edit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>420</y>
     <width>450</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: rgba(70, 60, 110, 0.8);
    color: #ffffff;
    border: 2px solid #6a5acd;
    border-radius: 22px;
    padding: 1px 150px 1px 50px;  /* جا برای آیکون */
    font-size: 18px;
}

QLineEdit:focus {
    border: 2px solid #9d81ff;
    background-color: rgba(90, 80, 140, 0.9);
}</string>
   </property>
   <property name="text">
    <string notr="true">Confirm New Password</string>
   </property>
   <property name="maxLength">
    <number>32765</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="pass_l2">
   <property name="geometry">
    <rect>
     <x>634</x>
     <y>425</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../store-management-requirment/lock_100dp_FFFFFF.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="pass_l1">
   <property name="geometry">
    <rect>
     <x>634</x>
     <y>346</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../store-management-requirment/lock_100dp_FFFFFF.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>310</y>
     <width>142</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 22px;
    font-weight: normal;
    color: #e0d4ff;
}</string>
   </property>
   <property name="text">
    <string>New Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="new_password_edit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>341</y>
     <width>450</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: rgba(70, 60, 110, 0.8);
    color: #ffffff;
    border: 2px solid #6a5acd;
    border-radius: 22px;
    padding: 1px 150px 1px 50px;  /* جا برای آیکون */
    font-size: 18px;
}

QLineEdit:focus {
    border: 2px solid #9d81ff;
    background-color: rgba(90, 80, 140, 0.9);
}</string>
   </property>
   <property name="text">
    <string>New Password</string>
   </property>
   <property name="maxLength">
    <number>32765</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>200</y>
     <width>271</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color : white 
}</string>
   </property>
   <property name="text">
    <string>Set New Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="security_label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../store-management-requirment/security_100dp_FFFFFF.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="back_login_Button">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>540</y>
     <width>134</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton[flat=&quot;true&quot;] {
    background: transparent;
    color: #d8bfff;
    border: none;
    font-size: 22px;
}

QPushButton[flat=&quot;true&quot;]:hover {
    color: #ffffff;
    text-decoration: underline;
}</string>
   </property>
   <property name="text">
    <string>Back to Login</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>reset_Button</zorder>
  <zorder>confirm_new_password_edit</zorder>
  <zorder>pass_l2</zorder>
  <zorder>label_6</zorder>
  <zorder>new_password_edit</zorder>
  <zorder>pass_l1</zorder>
  <zorder>label</zorder>
  <zorder>security_label</zorder>
  <zorder>back_login_Button</zorder>
 </widget>
 <tabstops>
  <tabstop>new_password_edit</tabstop>
  <tabstop>confirm_new_password_edit</tabstop>
  <tabstop>reset_Button</tabstop>
  <tabstop>back_login_Button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
